### PR TITLE
Wording changes (16 Jul 2024 ClearBank Changes)

### PR DESCRIPTION
**Pull request by Orkan-M**
PR to change wording for marketing team.

### CODE REVIEW

File: docs/10-index.md [hunk 1]
```diff
@@ -18,32 +15,21 @@ Replace:  Full Bank Name Limited
...
   - [Accounts & Transaction Information API](../swagger/account-info-openapi.yaml)
   - [Payments Initiation Services API](../swagger/payment-initiation-openapi.yaml)
-  - [Card Based Payment Instrument Issuer (CBPII) API](../swagger/confirmation-funds-openapi.yaml)
 
 ## Getting started
 
 We recommend that you start off by accessing our [Sandbox](./docs/40-sandbox.md).
 
-Our Sandbox fully reflects our production environment and provides an easy route to testing out your proposition.
+Our ClearBank’s Sandbox powered by ozone fully reflects our the production environment and provides an easy route to testing out your proposition.
```
Comment (lscancb) [inline docs/10-index.md:32]:
```suggestion
Our ClearBank Sandbox powered by ozone fully reflects the production environment and provides an easy route to testing out your proposition.
```

File: docs/20-getting-started.md [hunk 2]
```diff
@@ -2,14 +2,13 @@
 
 ## Dynamic Client Registration
 
-Our API allows dynamic client registration in order to create a valid client that is able to use our Authorisation Server. We only trust Software Statement Assertions (SSAs) issued by the Open Banking Directory provided by OBIE. eIDAS certificates are supported via onboarding to the Open Banking Directory (as discussed in more detail below).
+ClearBank’s API powered by Ozone API allows dynamic client registration in order to create a valid client that is able to use our Authorisation Server. We only trust Software Statement Assertions (SSAs) issued by the Open Banking Directory provided by OBIE. eIDAS certificates are supported via onboarding to the Open Banking Directory (as discussed in more detail below).
```
Comment (lscancb) [inline docs/20-getting-started.md:5]:
```suggestion
ClearBank’s API powered by Ozone allows dynamic client registration in order to create a valid client that is able to use our Authorisation Server. We only trust Software Statement Assertions (SSAs) issued by the Open Banking Directory provided by OBIE. eIDAS certificates are supported via onboarding to the Open Banking Directory (as discussed in more detail below).
```

File: docs/40-sandbox.md [hunk 3]
```diff
@@ -15,12 +15,12 @@ We provide a regulatory sandbox that fully reflects our production APIs.
 ### Step by Step guide to connect to sandbox
 
 #### Step 1: Pre-Requisites for TPP
-Ensure that the following pre-requisites are met before onboarding onto Clear Bank .
-The TPP has registered on the Directory Sandbox - https://directory.openbanking.org.uk/s/login/
-The TPP has at least one software statement created on the Directory Sandbox environment
-The TPP has at least one transport certificate created for each of its software statements.
-The TPP has at least one redirect URI for each of its software statements.
-The TPP has a copy of the OB root and issuing certificate attached.
+Ensure that the following pre-requisites are met before onboarding onto ClearBank.
```
Comment (lscancb) [inline docs/40-sandbox.md:18]:
```suggestion
Ensure that the following pre-requisites are met before onboarding with ClearBank.
```

File: docs/40-sandbox.md [hunk 4]
```diff
@@ -29,7 +29,8 @@ Please contact our Open Banking support team to be onboarded.
 
 #### Step 2: Register TPP with Bank using Dynamic Client Registration
 See http://openid.net/specs/openid-connect-registration-1_0-21.html
-Claims required in dynamic client registration for Banks are as follows
+
+Please find here a list of cClaims required in dynamic client registration for Banks:
```
Comment (lscancb) [inline docs/40-sandbox.md:33]:
```suggestion
Please find here a list of Claims required in dynamic client registration for Banks:
```

File: docs/40-sandbox.md [hunk 5]
```diff
@@ -122,7 +123,7 @@ You should now have successfully onboarded your TPP with the ASPSP and received
 
 2.3 Environment File download
 
-Once a TPP successfully registers to Clear Bank , they can download the environment file for their client via an Clear Bank  helper api
+Once a TPP successfully registers to ClearBank , they can download the environment file for their client via an ClearBank  helper api
```
Comment (lscancb) [inline docs/40-sandbox.md:126]:
```suggestion
Once a TPP successfully registers with ClearBank , they can download the environment file for their client via a supporting ClearBank api
```

File: docs/40-sandbox.md [hunk 6]
```diff
@@ -185,7 +186,7 @@ Select accounts
 
 ![](/assets/images/consent3.png)
 
-Once the PSU consent is successful, Clear Bank  will redirect back to the redirect URI. Copy the Authcode from the URL
+Once the PSU consent is successful, ClearBank  will redirect back to the redirect URI. Copy the Authcode from the URL
```
Comment (lscancb) [inline docs/40-sandbox.md:189]:
```suggestion
Once the PSU consent is successful, ClearBank will redirect back to the redirect URI. Copy the Authcode from the URL
```

File: docs/10-index.md (hunk 1 above)
Comment (mattregis) [inline docs/10-index.md:32]: tiny typo?

File: docs/API Overview/ais.md [hunk 7]
```diff
@@ -10,58 +10,30 @@ The base URL for all AIS APIs is: `https://rs1.{prod_domain_name}/open-banking/v
 [Accounts API](/perry/developer/documentation?resource=ukhub-clrb-portal&document=swagger/account-info-openapi.yaml#operations-tag-Accounts)
 
 Clear Bank  members will have one or more of the following accounts:
```
Comment (lscancb) [inline docs/API Overview/ais.md:12]:
```suggestion
ClearBank members will have one or more of the following accounts:
```

File: docs/API Overview/ais.md [hunk 8]
```diff
@@ -10,58 +10,30 @@ The base URL for all AIS APIs is: `https://rs1.{prod_domain_name}/open-banking/v
...
+- Safeguarding Account
 
-## Balances
-[Balances API](/perry/developer/documentation?resource=ukhub-clrb-portal&document=swagger/account-info-openapi.yaml#operations-tag-Balances)
-
-Balances shown in this endpoint provide the `InterimAvailable` value.
+## Balance
+The [Balances API](/perry/developer/documentation?resource=ukhub-clrb-portal&document=swagger/account-info-openapi.yaml#operations-tag-Balances) endpoint shows the account balance value. 
 
 `InterimAvailable` balance is the value displayed most widely to our members within the Clear Bank  apps.
```
Comment (lscancb) [inline docs/API Overview/ais.md:23]:
```suggestion
`InterimAvailable` balance is the value displayed most widely to our members within the ClearBank apps.
```

File: docs/40-sandbox.md (hunk 4 above)
Comment (mattregis) [inline docs/40-sandbox.md:33]: Claims?

File: docs/API Overview/pis.md [hunk 9]
```diff
@@ -1,29 +1,29 @@
...
+ClearBank's Open Banking API powered by Ozone **does not** support:
+- International Payments
+- File & Bulk Payments
 - Domestic Scheduled Payments
 - Domestic Standing Orders
 
-The payment request from TPP must have the creditor information.
 ## Overview
-The following apply to all domestic payment consents:
+The following fiels are mandatory fields for all domestic payment consents:
```
Comment (lscancb) [inline docs/API Overview/pis.md:20]:
```suggestion
The following fields are mandatory for all domestic payment consents:
```

Comment (mattregis) [inline docs/API Overview/pis.md:20]: files?

File: docs/API Overview/pis.md [hunk 10]
```diff
@@ -38,18 +38,54 @@ The following apply to all domestic payment consents:
...
-The PISP may also opt to populate reference field on behalf of the PSU
+The PISP may also opt to populate reference field on behalf of the PSU.
 
 ### `CreditorAccount`
-<!-- theme: info -->
-> ### Note
->
-> Domestic-payment-consents will only be authorised if the `CreditorAccount` details are sent received
-> If a consent contains recipient info that does not meet this criteria then error will be returned to the front end, the consent will remain in status `AwaitingAuthorisation` and the PSU will not be redirected.
+Domestic-payment-consents will only be authorised if the `CreditorAccount` details are sent or received. If a consent contains recipient info that does not meet this criteria then error will be returned to the front end, the consent will remain in status `AwaitingAuthorisation` and the PSU will not be redirected.
```
Comment (lscancb) [inline docs/API Overview/pis.md:44]:
```suggestion
Domestic-payment-consents will only be authorised if the `CreditorAccount` details are sent or received. If a consent contains recipient info that does not meet this criteria then an error will be returned to the front end, the consent will remain in status `AwaitingAuthorisation` and the PSU will not be redirected.
```

**Review by mattregis — APPROVED** [review]

**Review by lscancb — APPROVED** [review]

**Pull request merged**